migrations/2023-06-23-105727_alter_payouts/up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE payouts DROP COLUMN payout_method_data;
```

Comment:
```suggestion
ALTER TABLE payouts
ADD
    COLUMN payout_method_data JSONB DEFAULT '{}':: JSONB;
```

Reply:
This is the requirement -

UP - remove payout_method_data column since we will use locker to store the data
DROP - add back the payout_method_data column